monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```diff
@@ -344,7 +344,7 @@ local addStyle(alias, colorMode, colors, dateFormat, decimals, mappingType, patt
     local RbdDetailsPanel(title, formatY1, expr1, expr2, x, y, w, h) =
       graphPanelSchema({}, title, '', 'null as zero', false, formatY1, formatY1, null, null, 0, 1, '$Datasource')
       .addTargets(
-        [addTargetSchema(expr1, 1, 'time_series', 'Write'),addTargetSchema(expr2, 1, 'time_series', 'Read')]) + {gridPos: {x: x, y: y, w: w, h: h}};
+        [addTargetSchema(expr1, 1, 'time_series', '{{pool}} Write'),addTargetSchema(expr2, 1, 'time_series', '{{pool}} Read')]) + {gridPos: {x: x, y: y, w: w, h: h}};
```

Comment:
I added this because it conflicts with `rbd-overview.json`. If I understood it correctly, rbd-details provides metrics for each rbd pool so I thought that {{pool}} would be more specific and therefore make this work? @aaSharma14 you know more about this, what do you think.

Reply:
@aaSharma14 ping